components/messageBox/messageBox.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes } from 'react';
+import { getClassNamesWithMods, getDataAttributes } from '../_helpers';
+
+const MessageBox = (props) => {
+  const { children, dataAttrs, isError } = props;
+  const mods = props.mods ? props.mods.slice() : [];
```

Comment:
in my opinion, it looks much better: 
`const mods = getClassNamesWithMods('ui-messageBox', props.mods, { error: isError });`
instead of 
```
const mods = props.mods ? props.mods.slice() : [];
if (isError) {
   mods.push(`error`);
}
```